AXIOM_Remote_Firmware_Visualizer/VirtualUI.cpp

```diff
@@ -24,8 +24,11 @@ uint8_t lcdBrightness = 100;
 float lcdContrast = 1.0;
 bool toggleContrast = false;
 
-VirtualUI::VirtualUI(SDL_Window* window, uint32_t displayTextureID, CentralDB* db) :
-    _window(window), _io(ImGui::GetIO()), _displayTextureID(reinterpret_cast<ImTextureID>(displayTextureID)), _db(db)
+VirtualUI::VirtualUI(SDL_Window* window, uint32_t displayTextureID, CentralDB* db,
+                     std::function<void()> ScreenshotHandler) :
+    _window(window),
+    _io(ImGui::GetIO()), _displayTextureID(reinterpret_cast<ImTextureID>(displayTextureID)), _db(db),
+    _ScreenshotHandler(ScreenshotHandler)
```

Comment:
_screenshotHandler and screenshotHandler is more compliant for variables, arguments and class members.

Reply:
My thinking was that since it's a function, it should be capitalized.
I guess it is closer to being a class member than a class method though.

Reply:
Yes, you've described exactly why it's usually named like that.